### DISCUSSION TITLE
Invalid response: Found when trying to transfer database

### DISCUSSION THREAD

**Discussion by iBobik** [contember/admin#524] [Support] [answered]
What could be wrong? (Instead of better error message ;-))

```
> npm run contember data:transfer contember://aaa:xxxx@aaa.eu.contember.cloud --include-system

> contember
> docker-compose run --rm contember-cli data:transfer contember://aaa:xxxx@aaa.eu.contember.cloud

[+] Building 0.0s (0/0)                                                                           
[+] Creating 2/0
 ✔ Container aaa-postgres-1          Running                                            0.0s 
 ✔ Container aaa-contember-engine-1  Running                                            0.0s 
[+] Running 1/1
 ✔ Container aaa-postgres-1  Healthy                                                    0.5s 
[+] Building 0.0s (0/0)                                                                           

Transferring data between projects:

Source project name: aaa
Source API URL: https://aaa.eu.contember.cloud
Source token: xx***xx

Target project name: aaa
Target API URL: http://contember-engine:4000
Target token: 000***000

This will completely wipe the target project.
(to skip this dialog, you can pass --yes option)

✔ Do you want to continue? … yes
Error: Invalid response: Found

    at ClientRequest.<anonymous> (/opt/contember/run.js:101134:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Comment by honzasladek** [comment, marked as answer]
Hi @iBobik, we definitely should work on better err message / some validation. Anyway looking at your example I think there's the `api-` prefix missing. Can you please check? 🙏

**Reply by iBobik** [reply]
Aha, I expected I can use the same DSN like for deploy command.

**Reply by honzasladek** [reply]
Yes, happened to me as well. One possible solution is to proxy the requests made to non `api-` version. However I don't like that the proxy can bring some network issues. Probably will just add warning if `api-` is not there until we figure out a better way to solve this.

**Reply by iBobik** [reply]
I think CLI should add it there when parsing DSN. All commands should use unified DSN format, specific API endpoint is technical detail user should not care about.

**Reply by matej21** [reply]
You're right, it could be solved in the CLI. However, I'm not inclined to add hardcoded cloud-specific IFs into the open-source part of Contember.
